server/src/test/java/org/cloudfoundry/identity/uaa/provider/oauth/ExternalOAuthAuthenticationManagerTest.java

```diff
@@ -135,6 +135,7 @@ public void setup() throws Exception {
             entry(GROUP_ATTRIBUTE_NAME, "roles")
         );
         oidcConfig.setAttributeMappings(externalGroupMapping);
+        oidcConfig.setAdditionalAuthzParameters(Map.of("token_format", "jwt"));
```

Comment:
Can you please add a test case here that checks whether the additional authz parameters are properly added to the call to the IdP?